apps-rendering/src/components/series.tsx

```diff
@@ -49,18 +55,34 @@ const font = (isLabs: boolean): string =>
 		? textSans.medium({ lineHeight: 'loose', fontWeight: 'bold' })
 		: headline.xxxsmall({ lineHeight: 'loose', fontWeight: 'bold' });
 
-const linkStyles = (
-	{ kicker, inverted }: ThemeStyles,
+const getLinkColour = (
+	{ liveblogKicker, kicker }: ThemeStyles,
 	isLabs: boolean,
-): SerializedStyles => css`
-	${font(isLabs)}
-	color: ${isLabs ? palette.labs[300] : kicker};
-	text-decoration: none;
+	isBlog: boolean,
+): string => {
+	if (isLabs) {
+		return palette.labs[300];
+	}
+	if (isBlog) {
+		return liveblogKicker;
+	}
+	return kicker;
+};
 
-	${darkModeCss`
-        color: ${inverted};
-    `}
-`;
+const linkStyles = (
+	themeStyles: ThemeStyles,
+	isLabs: boolean,
+	isBlog: boolean,
+): SerializedStyles =>
+	css`
+		${font(isLabs)}
+		color: ${getLinkColour(themeStyles, isLabs, isBlog)};
+		text-decoration: none;
+
+		${darkModeCss`
+			color: ${isBlog ? themeStyles.liveblogKicker : themeStyles.inverted};
```

Comment:
I think the dark mode for blog doesn't have the correct color. It should be news[550] but it's using themeStyles.liveblogKicker which is news[600]